download_and_install/download_and_install.md

```diff
@@ -0,0 +1,53 @@
+Download and Install
+====================
+
+Pre-built binaries
+------------------
+
+OLA is available pre-built on many platforms:
+
+### Debian/Ubuntu (including Raspberry Pi)
+
+Raspberry Pi users may want to see the [Raspberry Pi Quick-Start Guide](ola_on_raspberry_pi)
+
+Ubuntu users will need the `universe` repository enabled.
+See [here](https://help.ubuntu.com/community/Repositories/Ubuntu) for details.
+
+    sudo apt-get update
+    sudo apt-get install ola
+
+### OpenSUSE
+
+Available from [multimedia:libs](https://build.opensuse.org/package/show/multimedia:libs/ola):
+
+    sudo zypper addrepo https://download.opensuse.org/repositories/multimedia:/libs/<VERSION> multimedia_libs
+    sudo zypper install ola
+
+If this is the first time software is installed from multimedia:libs, you may need to accept their package signing key.
+
+### Gentoo
+
+Gentoo packages are in progress [here](https://github.com/gentoo/gentoo/pull/19420).
+
+### macOS
+
+Available from [MacPorts](https://ports.macports.org/port/ola/summary):
+
+    sudo port install ola
+
+or [Homebrew](https://formulae.brew.sh/formula/ola#default):
+
+    brew install ola
+
+Build from source
+-----------------
+
+To stay on the bleeding edge with new and exciting features, [compile from source](compiling_from_source)
+
+Windows
+-------
+
+OLA is not presently available directly on Windows. However, it can be used inside a virtual machine. Because of the
+overhead of virtualization, there may be a small performance penalty. For show-critical uses, consider using a Linux
```

Comment:
This is now a bit out of date, some plugins are working on Windows:
https://github.com/OpenLightingProject/ola/blob/master/README.mingw32

List of plugins on the frontpage:
https://www.openlighting.org/ola/

Reply:
That's neat, I'll have a look tomorrow!

It seems the README.minigw32 disagrees with the openlighting.org page on working plugins - any idea which should be considered authoritative?

Reply:
Most likely the readme (or whichever lists the most stuff).

Every so often I go and do a test compilation on Windows and fix stuff if I can, or keep existing stuff going and then update the readme to match the compilation line I got working if I managed to add new stuff.